.circleci/config.yml

```diff
@@ -1,4 +1,17 @@
 version: 2.1
```

Comment:
can we just delete this file? we don't want to have anything related to circle CI left

Reply:
deleted